server/src/main/kotlin/net/horizonsend/ion/server/features/custom/items/blasters/BlasterListeners.kt

```diff
@@ -71,17 +73,31 @@ class BlasterListeners : SLEventListener() {
 	@EventHandler
 	fun onPlayerItemHoldEvent(event: PlayerItemHeldEvent) {
 		val itemStack = event.player.inventory.getItem(event.newSlot) ?: return
-		val customItem = itemStack.customItem as? Blaster<*> ?: return
+		val customItem = itemStack.customItem as? AmmunitionHoldingItem ?: return
```

Comment:
Not sure we want to extend this to magazines as well, best to move it to a common interface that contains the necessary values / methods